code/datums/components/crafting/recipes.dm

```diff
@@ -475,15 +475,28 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_AMMO
 
-/datum/crafting_recipe/ishotgun
-	name = "Improvised Shotgun"
-	result = /obj/item/gun/ballistic/shotgun/doublebarrel/improvised
+/datum/crafting_recipe/musket
+	name = "Maintenance musket"
+	result = /obj/item/gun/ballistic/rifle/boltaction/musket
 	reqs = list(/obj/item/weaponcrafting/receiver = 1,
 				/obj/item/pipe = 1,
 				/obj/item/weaponcrafting/stock = 1,
 				/obj/item/stack/package_wrap = 5)
 	tools = list(TOOL_SCREWDRIVER)
-	time = 100
+	time = 50
+	category = CAT_WEAPONRY
+	subcategory = CAT_WEAPON
+
+/datum/crafting_recipe/musket_prime
+	name = "Gray Maintenance musket"
+	result = /obj/item/gun/ballistic/rifle/boltaction/musket/prime
+	reqs = list(/obj/item/gun/ballistic/rifle/boltaction/musket = 1,
+				/obj/item/food/deadmouse = 1,
+				/datum/reagent/consumable/grey_bull = 20,
+				/obj/item/spear = 1,
+				/obj/item/storage/toolbox= 1)
```

Comment:
I'm not really sure why these (other than maybe the grey bull) are consumed when the musket is upgraded; aren't they foci to attune the gun to, like the insuls, gas mask, and cattleprod in the tools list, and not things that are actually added/strapped to the gun?

Reply:
Greyman guncraft is a mystery to all, even me.